### ISSUE TITLE
Maven artifact not found

### ISSUE THREAD

**Issue by d3btech** [EsotericSoftware/kryo#1045]
As specified in the readme, the following maven artifact could not be found by the IDE

```
<dependency>
   <groupId>com.esotericsoftware.kryo</groupId>
   <artifactId>kryo5</artifactId>
   <version>5.6.0</version>
</dependency>
```

**Comment by MrNavaStar** [comment]
Yes the maven is wrong in the read me, remove the `.kyro` from the group id

**Comment by d3btech** [comment]
@MrNavaStar Thanks. Also it was recommended to use `kryo5` for libraries. Is that correct information? We also need `objenesis` btw.

**Comment by theigl** [comment]
Thank you for reporting this @d3btech. 

I recently changed the structure of the projects Maven build and I seem to have broken something. I'll investigate.

The next release will have the correct Maven coordinates.

**Comment by theigl** [comment]
Should be resolved and I'll do a 5.6.1 release later this week.
